finalfrontier/src/vocab.rs

```diff
@@ -396,16 +420,49 @@ where
 {
     fn from(builder: VocabBuilder<SubwordVocabConfig, T>) -> Self {
         let config = builder.config;
+        let vocab_cutoff = config.vocab_cutoff;
```

Comment:
Factor out overlapping code with From impl for SimpleVocabConfig.

Helpful in refactoring: I don't think we are actually using EOS. IIRC I evaluated use of EOS at some point and there was no difference. Not sure why it shows up here and not above.

Reply:
@sebpuetz do you remember this?

Reply:
No recollection of any of that, but it seems about right that EOS marking doesn't have an effect considering we're training with punctuation.

Reply:
I found it odd too. Okay then, I guess I'll just ignore the whole EOS part. Thanks for the info.